stopify/src/runtime/check-runtime-opts.ts

```diff
@@ -31,11 +33,14 @@ export function checkAndFillRuntimeOpts(value: Partial<RuntimeOpts>): RuntimeOpt
   });
 
   const opts: RuntimeOpts = {
-    filename: '',
     estimator: 'velocity',
     yieldInterval: 100,
     resampleInterval: 100,
     timePerElapsed: 1,
+    stackSize: Infinity,
+    restoreFrames: Infinity,
+
+    filename: '',
```

Comment:
Don't refactor! Please be disciplined about how you make changes. Undo this.

Reply:
I added two flags here and moved the ones that are only used for testing and benchmarking to the bottom.
Do you want me to minize the diffs to just the changes needed? Adding spacing helps me read code.